VerveAdapter/build.gradle.kts

```diff
@@ -73,7 +73,7 @@ dependencies {
     "remoteImplementation"("com.chartboost:chartboost-mediation-sdk:4.0.0")
 
     // Partner SDK
-    implementation("net.pubnative:hybid.sdk:2.17.0")
+    implementation("net.pubnative:hybid.sdk:2.19.0")
```

Comment:
As @chauduyphanvu mentioned, the version of this adapter will now be `4.2.19.0.0`. Let's update the CHANGELOG, README and VerveAdapter/build.gradle.kts to this new version.